frontend/src/features/dashboard/components/PlannedProductionModal.tsx

```diff
@@ -0,0 +1,367 @@
+                                        error={formError && !isValidOrderReference(temporaryProduction)}
+                                        value={temporaryProduction?.customerOrderNumber ?? ''}
+                                        onChange={(event) =>
+                                            setTemporaryProduction({ ...temporaryProduction, customerOrderNumber: event.target.value })
+                                        }
+                                    />
+                                </Grid>
+                                <Grid item xs={6}>
+                                    <Input
+                                        id="customer-order-position-number"
+                                        label="Customer Order Position"
+                                        type="text"
+                                        error={formError && !isValidOrderReference(temporaryProduction)}
+                                        value={temporaryProduction?.customerOrderPositionNumber ?? ''}
+                                        onChange={(event) =>
+                                            setTemporaryProduction({
+                                                ...temporaryProduction,
+                                                customerOrderPositionNumber: event.target.value,
+                                            })
+                                        }
+                                    />
+                                </Grid>
+                                <Grid item xs={6}>
+                                    <Input
+                                        id="supplier-order-number"
+                                        label="Supplier Order Number"
+                                        type="text"
+                                        error={formError && !isValidOrderReference(temporaryProduction)}
+                                        value={temporaryProduction?.supplierOrderNumber ?? ''}
+                                        onChange={(event) =>
+                                            setTemporaryProduction({
+                                                ...temporaryProduction,
+                                                supplierOrderNumber: event.target.value ?? '',
+                                            })
+                                        }
+                                    />
+                                </Grid>
+                            </>
+                        ) : (
+                            <Grid item xs={12}>
+                                <Table
+                                    title={`Planned Production ${
+                                        temporaryProduction?.estimatedTimeOfCompletion
+                                            ? ' on ' +
+                                              new Date(temporaryProduction?.estimatedTimeOfCompletion).toLocaleDateString('en-UK', {
```

Comment:
We're using a different localeDateString here right? This might be the cause of the issue described above.

Reply:
This was unrelated to the problem, but I unified the locale regardless to default to the user's locale